src/frontend/src/pages/ProjectPage/pages/ContractPage/pages/ManagePersonnelPage/AddPersonnelSideSheet/AddPersonnelFormHeader.tsx

```diff
@@ -6,10 +6,10 @@ type HeaderProps = {
   headers: string[];
 }
 
-const AddPersonnelFormHeader : React.FC<HeaderProps> = ({headers}) => {
+const AddPersonnelFormHeader: React.FC<HeaderProps> = ({ headers }) => {
   return (
     <>
-      <div className={classNames(styles.cell, styles.header, styles.expand)} >
+      <div className={classNames(styles.cell, styles.header)} >
```

Comment:
I think the entire table would be simpler if we used `<thead><tr><th>{headerLabel}</th><th>{headerLabel}</th><th>{headerLabel}</th></tr></thead>` instead of combining divs and a table